### PR TITLE
Expand inter-MITTEN-t logic to account for confirm-failures.

### PR DESCRIPTION
**Pull request by jmaher**

### CODE REVIEW

**Comment by jmaher** [comment]
if you run this locally, you can see examples via:
http://localhost:5000/jobs?repo=try&revision=92c66017622ae7f6524dbfc1ebc7cb6e03f64d2a

File: ui/job-view/pushes/JobsAndGroups.jsx [hunk 1]
```diff
@@ -18,6 +18,27 @@ export default class JobsAndGroups extends React.Component {
...
+        let gname = '';
+        for (let tier = 1; tier <= 3; tier++) {
+          gname = g.mapKey.replace('-cf3', tier);
+          for (const t of groups) {
+            if (t.mapKey === gname) foundTier = true;
+          }
+          if (foundTier) break;
+        }
+
+        if (foundTier) {
```
Comment (Archaeopteryx) [inline ui/job-view/pushes/JobsAndGroups.jsx:36]: This should merge with line 33 - just one `if (foundTier)` condition with the statements.

File: ui/job-view/pushes/JobsAndGroups.jsx [hunk 2]
```diff
@@ -18,6 +18,27 @@ export default class JobsAndGroups extends React.Component {
...
 
+    const confirmGroups = {};
+    for (const g of groups) {
+      // group.mapKey == pushID groupSymbol Tier platform buildtype
+      // find matching group.mapKey
+      if (g.symbol.endsWith('-cf')) {
+        let foundTier = false;
+        let gname = '';
+        for (let tier = 1; tier <= 3; tier++) {
+          gname = g.mapKey.replace('-cf3', tier);
```
Comment (Archaeopteryx) [inline ui/job-view/pushes/JobsAndGroups.jsx:29]: Could you explain where and how the `-cf3` gets created?

**Review by Archaeopteryx — APPROVED** [review]

File: ui/job-view/pushes/JobsAndGroups.jsx (hunk 2 above)
Comment (jmaher) [reply ui/job-view/pushes/JobsAndGroups.jsx:29]: as seen here: 
https://github.com/mozilla/treeherder/blob/1b0aa257f0d599e274d86dc3db6fa26d36bd26c0/ui/helpers/aggregateId.js#L17

this ends up squashing many items together to make a "unique" mapkey.

**Pull request merged**